backend/api/templates/api/graphiql.html

```diff
@@ -0,0 +1,41 @@
+{% load static %}
```

Comment:
Shall we use this? https://github.com/OneGraph/graphiql-explorer :D

Reply:
can be done in another PR :)

Reply:
I will do it in another PR as we want to merge this ASAP :)